packages/grafana-llm-app/pkg/main.go

```diff
@@ -8,8 +8,14 @@ import (
 	"github.com/grafana/grafana-plugin-sdk-go/backend/log"
 )
 
+// version is the version of the plugin.
+// It is set at build time by the Mage target, which passes
+// the version to the Go linker using `-X main.version=x.y.z`.
+var version = "development"
+
 func main() {
-	log.DefaultLogger.Info("Starting plugin process")
+	plugin.PluginVersion = version
```

Comment:
should this be done in an `init()` next to the variable definition instead?

Reply:
I guess we can't really due to the dependency order. I guess this is fine, though maybe `init()` is still more correct than `main()`.

Reply:
Yup that's better, thanks.